wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/CompleteReactionPicker/CompleteReactionPickerViewController.swift

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+class CompleteReactionPickerViewController: UIViewController {
+
+    weak var delegate: EmojiPickerViewControllerDelegate?
+    private var emojiDataSource: EmojiDataSource!
+    private let collectionView = ReactionsCollectionView()
+    private  let sectionViewController = ReactionSectionViewController(types: EmojiSectionType.all)
+    private let topBar = ModalTopBar()
+    private let searchBar = UISearchBar()
+
+    private var deleting = false
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+
+        emojiDataSource = EmojiDataSource(provider: cellForEmoji)
+        collectionView.dataSource = emojiDataSource
+        collectionView.delegate = self
+        searchBar.delegate = self
+        sectionViewController.sectionDelegate = self
+        setupViews()
+        createConstraints()
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        collectionView.reloadItems(at: collectionView.indexPathsForVisibleItems)
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        updateSectionSelection()
+    }
+
+    func setupViews() {
+        typealias Strings = L10n.Localizable.Content.Reactions
+        view.addSubview(topBar)
+        topBar.delegate = self
+        topBar.needsSeparator = false
+        topBar.backgroundColor = SemanticColors.View.backgroundDefault
+        topBar.configure(title: Strings.title, subtitle: nil, topAnchor: safeTopAnchor)
+
+        addChild(sectionViewController)
+        view.addSubview(sectionViewController.view)
+        sectionViewController.didMove(toParent: self)
+
+        searchBar.backgroundImage = UIImage()
+        searchBar.searchTextField.backgroundColor = SemanticColors.View.backgroundDefaultWhite
+        searchBar.placeholder = Strings.search
+        view.addSubview(searchBar)
+        view.backgroundColor = SemanticColors.View.backgroundDefault
+        view.addSubview(collectionView)
+    }
+
+    private func createConstraints() {
+        guard let sectionViewControllerView = sectionViewController.view else { return }
```

Comment:
why do we need this guard ? view is not optional. Did you mean?
```suggestion
        guard let sectionViewControllerView = sectionViewController. viewIfLoaded else { return }
```

Reply:
![image](https://github.com/wireapp/wire-ios-mono/assets/1401672/e2578726-0ae6-46ba-b346-96827151c467)
it 's throwing error when there's no guard